accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/generators/DecoratedAccessionGeneratorTest.java

```diff
@@ -128,14 +132,17 @@ public void testGenerateSuffix() throws Exception {
 
         generator.postSave(new SaveResponse<>(savedAccessions, new HashSet<>()));
         assertEquals(2, repository.findFirstByCategoryIdAndApplicationInstanceIdOrderByLastValueDesc(
-                CATEGORY_ID, INSTANCE_ID).getLastCommitted());
+                monotonicGeneratorProperties.getCategoryId(), monotonicGeneratorProperties.getApplicationInstanceId())
+                .getLastCommitted());
     }
 
     private MonotonicAccessionGenerator<String> getGenerator() throws Exception {
         assertEquals(0, repository.count());
 
-        MonotonicAccessionGenerator<String> generator = new MonotonicAccessionGenerator<>(BLOCK_SIZE, CATEGORY_ID,
-                INSTANCE_ID, service);
+        MonotonicAccessionGenerator<String> generator = new MonotonicAccessionGenerator<>
```

Comment:
Again, no, this is not what I meant about check how the init value configuration is passed to the block service. You should end with less configuration parameters here, not more.